### PR TITLE
MRG: Unify axis labeling

### LINKED ISSUE #4534: Capitalization in axis labels
Hey guys 
amazing stuff your are doing ...
just wondered if it would be possible to switch the default for axis labeles to first letter capitalized (e.g. from time [ms] to Time [ms])? 

Would be great as it is a requirement for several journals ... (Nature: Lettering in figures (labelling of axes and so on) should be in lower-case type, with the first letter capitalized and no full stop.)

Cheers
Ben

### PR DESCRIPTION
**Pull request by larsoner**
Closes #4534.

This PR:

- Unifies axis labeling to "Time (s)" form from others like "time [sec]". Turns out there were a lot of places we already used the former standard so the changeset wasn't so bad.
- Unifies handling of unit naming in ICA property plots to fit with e.g. `raw.plot_psd()`.
- Unifies `sphx_glr_*`-based linking rather than custom `tut_` ones to make things more standard.

### CODE REVIEW

File: mne/viz/epochs.py [hunk 1]
```diff
@@ -911,10 +911,8 @@ def plot_epochs_psd(epochs, fmin=0, fmax=np.inf, tmin=None, tmax=None,
                             color=color, alpha=area_alpha)
         if make_label:
             if ii == len(picks_list) - 1:
-                ax.set_xlabel('Freq (Hz)')
-            ax.set_ylabel(ylabel)
-            ax.set_title(title)
-            ax.set_xlim(freqs[0], freqs[-1])
+                ax.set_xlabel('Frequency (Hz)')
+            ax.set(ylabel=ylabel, title=title, xlim=(freqs[0], freqs[-1]))
```
Comment (jona-sassenhagen) [inline mne/viz/epochs.py]: What, this is a thing? I never knew!

File: mne/viz/evoked.py [hunk 2]
```diff
@@ -269,8 +269,7 @@ def _plot_evoked(evoked, picks, exclude, unit, show, ylim, proj, xlim, hline,
                     units, scalings, hline, gfp, types, zorder, xlim, ylim,
                     times, bad_ch_idx, titles, ch_types_used, selectable,
                     False, line_alpha=1.)
-        for ax in axes:
-            ax.set_xlabel('Time (ms)')
+        plt.setp(axes, xlabel='Time (ms)')
```
Comment (agramfort) [inline mne/viz/evoked.py]: you want to change all these to 'Time (s)' right?

File: mne/viz/ica.py [hunk 3]
```diff
@@ -652,10 +648,7 @@ def _plot_ica_overlay_raw(data, data_cln, times, title, ch_types_used, show):
     plt.suptitle(title)
     ax1.plot(times, data.T, color='r')
     ax1.plot(times, data_cln.T, color='k')
-    ax1.set_xlabel('time (s)')
-    ax1.set_xlim(times[0], times[-1])
-    ax1.set_xlim(times[0], times[-1])
```
Comment (jona-sassenhagen) [inline mne/viz/ica.py]: ... what

File: mne/viz/misc.py [hunk 4]
```diff
@@ -769,12 +765,12 @@ def plot_filter(h, sfreq, freq=None, gain=None, title=None, color='#1f77b4',
     axes[1].set(ylabel='Magnitude (dB)', xlabel='', xscale=fscale)
     sl = slice(0 if fscale == 'linear' else 1, None, None)
     axes[2].plot(f[sl], gd[sl], color=color, linewidth=2, zorder=4)
-    axes[2].set(xlim=flim, ylabel='Group delay (sec)', xlabel='Frequency (Hz)',
+    axes[2].set(xlim=flim, ylabel='Group delay (s)', xlabel='Frequency (Hz)',
```
Comment (jona-sassenhagen) [inline mne/viz/misc.py]: Maybe it would be nicer if you did `fig, (time_ax, freq_ax, delay_ax) = plt.subplots(3, squeeze=True)` in 754.

**Comment by jona-sassenhagen** [comment]
LGTM

File: mne/viz/misc.py [hunk 5]
```diff
@@ -760,7 +756,7 @@ def plot_filter(h, sfreq, freq=None, gain=None, title=None, color='#1f77b4',
     f = omega * sfreq / (2 * np.pi)
     axes[0].plot(t, h, color=color)
     axes[0].set(xlim=t[[0, -1]], xlabel='Time (sec)',
```
Comment (jona-sassenhagen) [inline mne/viz/misc.py]: Time (s)

File: mne/viz/evoked.py (hunk 2 above)
Comment (larsoner) [reply mne/viz/evoked.py]: In #3164 this is discussed, yes. It's not implemented here, though, and would be better as a follow-up PR if people agree we should do it.

File: mne/viz/ica.py (hunk 3 above)
Comment (larsoner) [reply mne/viz/ica.py]: agreed :)

**Comment by larsoner** [comment]
Rendered ICA ex:

https://7567-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_artifacts_correction_ica.html

File: mne/viz/misc.py [hunk 6]
```diff
@@ -599,9 +597,7 @@ def plot_dipole_amplitudes(dipoles, colors=None, show=True):
         ax.plot(dip.times, dip.amplitude * 1e9, color=color, linewidth=1.5)
         xlim[0] = min(xlim[0], dip.times[0])
         xlim[1] = max(xlim[1], dip.times[-1])
-    ax.set_xlim(xlim)
-    ax.set_xlabel('Time (sec)')
-    ax.set_ylabel('Amplitude (nAm)')
+    ax.set(xlim=xlim, xlabel='Time (sec)', ylabel='Amplitude (nAm)')
```
Comment (jona-sassenhagen) [inline mne/viz/misc.py]: (sec)?

**Comment by agramfort** [comment]
thx @larsoner

**Pull request merged**